components/HTTP/src/Websocket.cpp

```diff
@@ -367,14 +366,14 @@ void Websocket::websocketRadioRx(const std::uint8_t* buffer, std::size_t length)
 {
     chat_msg_t msg;
     esp_err_t err;
-    size_t uidlen = 32;
     err = message::parseMessage((std::uint8_t*)buffer, &msg, length);
     if (err) {
         ESP_LOGE("TEST", "error decoding the value");
     }
 
     cJSON* root = cJSON_CreateObject();
 
+    size_t uidlen = 32;
     char fromUID[65];
     util::bytesToHex(msg.from_uid, fromUID, uidlen);
```

Comment:
```suggestion
    char fromUID[65];
    util::bytesToHex(msg.from_uid, fromUID, sizeof(msg.from_uid));
```
This avoids declaring another variable, just as a side note: we should convert `chat_id_t` to a `class ChatId` with `from_hex` constructors or something more idiomatic in C++, can be done in another PR.

Reply:
And overload the `operator ==` to safely compare chat ids like `a == b` instead of using `memcmp`.